### PR TITLE
Remove default sensor_types, convert tabs to spaces

### PR DESCRIPTION
**Pull request by AndrewEllis93**
- Remove default `sensor_type:` values to ensure that users are verifying their thermistors. Many users are just using the default values without actually checking.
- Add comments linking to Klipper thermistor list (so always up-to-date)
- Remove thermistor list at bottom of config files (if it exists - better to use the always up-to-date Klipper link)
- Hint at using "Generic 3950" for NTC 100k 3950, and also for Keenovo heaters
- Remove custom 3950 definition (if exists) - no longer necessary, Klipper thermistor accuracy bug has been fixed, and any new builders will be pulling a new version of Klipper.

### CODE REVIEW

**Comment by AndrewEllis93** [comment]
- Converted all tabs to white spaces, preserving alignment, so it displays correctly in all editors and on GitHub
- Commented out sensor_types - this makes the error easier to understand (thanks for the idea Tetsunosuke)
image

File: firmware/klipper_configurations/Spider/Voron2_Spider_Config.cfg [hunk 1]
```diff
@@ -510,14 +513,4 @@ gcode:
...
-## 	Common Temperature Sensors
-##   "EPCOS 100K B57560G104F"
-##   "ATC Semitec 104GT-2"
-##   "NTC 100K beta 3950"
-##   "Honeywell 100K 135-104LAG-J01"
-##   "NTC 100K MGB18-104F39050L32" (Keenovo Heater Pad)
-##   "AD595"
-##   "PT100 INA826"
-##   "PT1000"
+    BED_MESH_CLEAR
```
Comment (cbc02009) [inline firmware/klipper_configurations/Spider/Voron2_Spider_Config.cfg]: Need a newline at the end of the file.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

**Comment by AndrewEllis93** [comment]
Ensured newlines at the end of each file.
Removed 3950 footnotes (if any)

**Comment by AndrewEllis93** [comment]
Side note: these EOF newlines don't seem to always display correctly in GitHub. When I download the raw files (not just view raw in browser), though, they are there.

**Review by cbc02009 — APPROVED** [review]

**Review by raymondh2 — APPROVED** [review]

**Pull request merged**